ksqldb-streams/src/test/java/io/confluent/ksql/execution/streams/SourceBuilderTest.java

```diff
@@ -768,6 +849,20 @@ private void givenWindowedSourceTable() {
     );
   }
 
+  private void givenWindowedMultiKeySourceTable() {
```

Comment:
It's not possible for a user to have a windowed, multi-key table today, right? Since group by on multiple columns still creates a single key, and we don't allow importing windowed tables?

Also, this method looks unused.

Reply:
yes you are right, that was from a change i had backed out and forgot to udpate! thanks for catching this